src/tests/multicityRequest.test.js

```diff
@@ -0,0 +1,57 @@
+
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import localStorage from 'localStorage';
+import app from '../app';
+import mockData from './mockData';
+
+chai.use(chaiHttp);
+chai.should();
+
+const multicityRequest = () => {
+  describe('Multicity request ', () => {
+    before((done) => {
+      chai
+        .request(app)
+        .post('/api/v1/auth/login')
+        .send(mockData.loginNewUser)
+        .end((err, res) => {
+          localStorage.setItem('token', res.body.token);
+          done();
+        });
+    });
+
+    it('should return 200 on successful created request ', (done) => {
+      chai
+        .request(app)
+        .post('/api/v1/trips/request/multicity')
+        .send(mockData.multiCityRequest)
+        .end((err, res) => {
+          expect(res.status).to.equal(201);
```

Comment:
@EugeneMunya  please include the message tests as well, this applies on the rest of the tests

Reply:
I tested also the messages, you see how I managed to do it